glambie/util/timeseries_helpers.py

```diff
@@ -238,6 +238,29 @@ def timeseries_as_months(fractional_year_array: np.array, downsample_to_month: b
     return monthly_array
 
 
+def timeseries_is_monthly_grid(fractional_year_array: np.array) -> bool:
```

Comment:
Ahh, I get it now. I can think of terms other than "monthly grid", but it does succesfully communicate that we're not talking about calendar months.